### PR TITLE
V2

### PR DESCRIPTION
**Pull request by galeksandrp**
All from scratch.
- [x] v2-22tracks
- [x] v2-61
- [x] v2-ambientsleepingpill
- [x] v2-aolradio
- [x] v2-baidu
- [ ] v2-beats
- [x] v2-blitzr
- [x] v2-daytrotter
- [x] v2-deezer
- [x] v2-difm
- [x] v2-douban-programme
- [x] v2-fizy
- [x] v2-focusatwill
- [x] v2-gaana (#751)
- [x] v2-groovemusic
- [x] v2-hillydilly (#771)
- [x] v2-jango (#772)
- [x] v2-letournedisque
- [x] v2-luoo
- [x] v2-megalyrics
- [x] v2-mojepolskieradio
- [x] v2-myspace
- [x] v2-novaplanet
- [x] v2-nrkradio
- [x] v2-odnoklassniki
- [x] v2-pakartot
- [x] v2-pitchfork
- [x] v2-pleer
- [x] v2-plex
- [x] v2-plugdj
- [x] v2-radioplayer
- [x] v2-radioplusbe
- [ ] v2-rdio
- [x] v2-redditplayer
- [x] v2-reverbnation
- [x] v2-soundozer
- [x] v2-sullen-ural
- [x] v2-ttnet
- [x] v2-tubafm
- [x] v2-tunein (#977)
- [x] v2-virginradiotr
- [x] v2-vk
- [x] v2-weborama
- [x] v2-yandex
- [x] v2-zvooq

### CODE REVIEW

**Comment by janbaykara** [comment]
Perhaps also delete the v1 connectors that are replaced?

File: connectors/v2/tunein.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+/* global Connector */
+
+Connector.playerSelector = '.container';
+
+Connector.artistTrackSelector = '.line1._navigateNowPlaying';
+
+Connector.isPlaying = function () {
+	return $('#tuner').hasClass('playing');
+};
```
Comment (Paszt) [inline connectors/v2/tunein.js]: lol, I was told each connector should be a separate pull request for ease of testing.  Anyway you could add artwork here

```
Connector.trackArtImageSelector = '.image img';
```

**Comment by fedorg** [comment]
The notification should have disable controls (e.g. #795)

**Comment by janbaykara** [comment]
So we just need to add & tick off v2-rdio and v2-beats, to get these done?

**Comment by fedorg** [comment]
Isn't V2-beats already done in #729?

**Comment by janbaykara** [comment]
Then @galeksandrp should tick it off in the opening post.

**Comment by Krishath** [comment]
I think working v2 connectors should be merged. I'd like to work on the v2 connector of Deezer (since it doesn't scrobble albums) but I'd like to have a "final" v2 version first.

**Comment by inverse** [comment]
Thanks for all your hardwork in this PR! @alexesprit broke it down and we got them all in one by one.

Cheers @galeksandrp
